packages/ckeditor5-emoji/src/utils/isemojisupported.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Copyright (c) 2023, Koala Interactive SAS
+ * For licensing, see https://github.com/koala-interactive/is-emoji-supported/blob/master/LICENSE.md
+ */
+
+/**
+ * Checks if the two pixels parts are the same using canvas.
+ */
+export default function isEmojiSupported( unicode: string ): boolean {
+	const ctx = getCanvas();
+
+	/* istanbul ignore next -- @preserve */
+	if ( !ctx ) {
+		return false;
+	}
+
+	const CANVAS_HEIGHT = 25;
+	const CANVAS_WIDTH = 20;
+	const textSize = Math.floor( CANVAS_HEIGHT / 2 );
+
+	// Initialize canvas context.
+	ctx.font = textSize + 'px Arial, Sans-Serif';
+	ctx.textBaseline = 'top';
+	ctx.canvas.width = CANVAS_WIDTH * 2;
+	ctx.canvas.height = CANVAS_HEIGHT;
+
+	ctx.clearRect( 0, 0, CANVAS_WIDTH * 2, CANVAS_HEIGHT );
+
+	// Draw in red on the left.
+	ctx.fillStyle = '#FF0000';
+	ctx.fillText( unicode, 0, 22 );
+
+	// Draw in blue on right.
+	ctx.fillStyle = '#0000FF';
+	ctx.fillText( unicode, CANVAS_WIDTH, 22 );
+
+	const a = ctx.getImageData( 0, 0, CANVAS_WIDTH, CANVAS_HEIGHT ).data;
+	const count = a.length;
+	let i = 0;
+
+	// Search the first visible pixel.
+	for ( ; i < count && !a[ i + 3 ]; i += 4 ) ;
```

Comment:
```suggestion
	for ( ; i < count && !a[ i + 3 ]; i += 4 );
```

Reply:
Fixed in [7343da3](https://github.com/ckeditor/ckeditor5/pull/17864/commits/7343da3b2734aaf3d93b6e1e8e455c9a2bf881df)